lib/modules/userTagger.js

```diff
@@ -361,7 +361,9 @@ modules['userTagger'] = {
 			this.applyTags();
 			//console.log('after applytags: ' + Date());
 			if (RESUtils.pageType() === 'comments') {
-				RESUtils.watchForElement('newComments', modules['userTagger'].attachVoteHandlers);
+				RESUtils.watchForElement('newComments', function(ele) {
+					modules['userTagger'].attachVoteHandlers(ele.parentNode);
```

Comment:
Replace this line with the following?

    modules['userTagger'].attachVoteHandlers(ele.parentNode.querySelector('.midcol'));

Might not be the most efficient way, though.